etc/ci/upload_nightly.sh

```diff
@@ -12,7 +12,12 @@ usage() {
 
 
 upload() {
-    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    local s3path filename
+    s3path="s3://servo-builds/nightly"
+    filename=test=$(basename "${2}")
+    
+    s3cmd put "${s3path}/${2}" "${s3path}/${1}/"
```

Comment:
Also, we might as well be explicit about the full path for the upload location (last argument) and include the filename. In the case, we probably want to pull that into a variable so it can be reused on the next line.